dask/array/fft.py

```diff
@@ -73,9 +92,17 @@ def _ihfft_out_chunks(a, n, axis):
 
 
 _out_chunk_fns = {'fft': _fft_out_chunks,
+                  'fft2': _fft_out_chunks,
+                  'fftn': _fft_out_chunks,
                   'ifft': _fft_out_chunks,
+                  'ifft2': _fft_out_chunks,
+                  'ifftn': _fft_out_chunks,
                   'rfft': _rfft_out_chunks,
+                  'rfft2': _rfft_out_chunks,
+                  'rfftn': _rfft_out_chunks,
                   'irfft': _irfft_out_chunks,
+                  'irfft2': _irfft_out_chunks,
+                  'irfftn': _irfft_out_chunks,
                   'hfft': _hfft_out_chunks,
                   'ihfft': _ihfft_out_chunks}
```

Comment:
It's a little less clear, but sure we could. Do you think these will actually change that much though?